packages/google_sign_in/google_sign_in_ios/darwin/Tests/GoogleSignInTests.m

```diff
@@ -92,78 +99,76 @@ - (void)testInitGoogleServiceInfoPlist {
                                                clientId:nil
                                          serverClientId:nil];
 
-  FlutterError *error;
-  [self.plugin initializeSignInWithParameters:params error:&error];
-  XCTAssertNil(error);
+  FlutterError *initializationError;
+  [self.plugin initializeSignInWithParameters:params error:&initializationError];
+  XCTAssertNil(initializationError);
 
   // Initialization values used in the next sign in request.
   [self.plugin signInWithCompletion:^(FSIUserData *user, FlutterError *error){
   }];
-  OCMVerify([self.mockSignIn
-       signInWithConfiguration:[OCMArg checkWithBlock:^BOOL(GIDConfiguration *configuration) {
-         // Set in example app GoogleService-Info.plist.
-         return
-             [configuration.hostedDomain isEqualToString:@"example.com"] &&
-             [configuration.clientID
-                 isEqualToString:
-                     @"479882132969-9i9aqik3jfjd7qhci1nqf0bm2g71rm1u.apps.googleusercontent.com"] &&
-             [configuration.serverClientID isEqualToString:@"YOUR_SERVER_CLIENT_ID"];
-       }]
-      presentingViewController:[OCMArg isKindOfClass:[FlutterViewController class]]
-                          hint:nil
-              additionalScopes:OCMOCK_ANY
-                      callback:OCMOCK_ANY]);
+  OCMVerify([self configureMock:self.mockSignIn
```

Comment:
As with the changes in the implementation file, all of these changes are whitespace changes from the switch to the new `configureMock` helper.